jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -203,8 +203,12 @@ def draw_compass_mpl(image, orig_shape=None, wcs=None, show=True, zoom_limits=No
         plt.ioff()
 
     fig, ax = plt.subplots()
+    if transform is not None:
+        transform += ax.transData
+    else:
+        transform = ax.transData
     ax.imshow(image, extent=[-0.5, orig_shape[1] - 0.5, -0.5, orig_shape[0] - 0.5],
-              origin='lower', cmap='gray', **kwargs)
+              origin='lower', cmap='gray', transform=transform + ax.transData, **kwargs)

```

Comment:
```suggestion
              origin='lower', cmap='gray', transform=transform, **kwargs)
```
... and apply the second suggestion to change parameters accordingly, which somehow was lost in the void
(again, why does GH not just allow one to put everything into one suggestion...? 🤯)

Reply:
Applied. Thanks!